pretixdesk/src/main/kotlin/eu/pretix/pretixdesk/ui/SetupView.kt

```diff
@@ -141,4 +172,11 @@ class SetupView : View() {
             handleConfiguration(event.rawUrl)
         }
     }
+
+    private fun icon(icon: MaterialIcon): MaterialIconView {
+        val iconView = MaterialIconView(icon)
+        iconView.glyphSize = 25
+        iconView.glyphStyle = "-fx-fill: white;"
```

Comment:
I don't know if this is the best way to do it... didn't understand how to do this with tornadofx's Kotlin styling syntax